src/issue.js

```diff
@@ -95,11 +121,16 @@ async function createOrUpdateIssue(
   registerHandlebarHelpers({
     commitMessageLines,
   })
+
+  const semVerReleaseType = await getAutoBumpedVersion()
+
   const issueBody = await renderIssueBody({
     commits: unreleasedCommits,
     latestRelease,
     notifyAfter,
+    semVerReleaseType,
```

Comment:
Using your updated action I don't get the new release version suggestion in the issue description:

<img width="911" alt="image" src="https://user-images.githubusercontent.com/6604111/215969077-2b3ffade-f15a-4330-b3cb-2b56266f9989.png">

This is because you are passing to `renderIssueBody` a key named `semVerReleaseType` but in the `issue.template.hbs` you are referencing to `newVersion`.